### DISCUSSION TITLE
Pending transaction waiting to close

### DISCUSSION THREAD

**Discussion by Nuggs86** [Ride-The-Lightning/RTL#1264] [Q&A] [answered]
Hey, I have tried to close a channel back in April and it is still in 'Waiting Close'.  All I can do is view info. How can I finalise this transaction. I think I have set the fee to low. I am very new to this so not entirely sure what I am doing. The transaction is mempool but is still unconfirmed. Help please!

**Comment by saubyk** [comment, marked as answer]
Hi @Nuggs86 if you tried to close it in April and it's still not confirmed, it's highly likely that the transaction has been dropped from the mempool. Can you confirm which lightning implementation are you running?

If it's LND, it tries to rebroadcast pending transactions automatically. Do you see an unconfirmed onchain transactions on the onchain page? You can try to CPFP the pending transaction to bump up the confirmation

**Reply by Nuggs86** [reply]
I am using raspiblitz, looks it has tried to rebroadcast yesterday. I can see the amount under unconfirmed on the onchain page. how can CPFP the transaction. Its just sitting in pending channels, Waiting Close. All I can do is view the info.

**Reply by saubyk** [reply]
You'll have to do it via the command line interface. You can use the bumpfee command to CPFP
Here's the relevant documentation:
https://lightning.engineering/api-docs/api/lnd/wallet-kit/bump-fee

**Reply by saubyk** [reply]
Keep in mind while bumping you should consider the prevailing fee rate via mempool.space
And bump it up with enough rate so that the average rate works out to be good enough for miners to prioritize, otherwise you'll be stuck in the same situation

**Reply by Nuggs86** [reply]
how do i get to the command line interface. I have basically no idea what i am doing and trying to learn. Thanks

**Reply by saubyk** [reply]
That question has to be addressed by the Raspiblitz team, I am not familiar with the interfaces enough to provide you guidance on it

What you are basically doing with CPFP is spending the unconfirmed output back to yourselves, but ensuring you pay a high enough fee rate that the transaction (including the parent) is juicy enough for the miners to mine

I would recommend reading up on it, to be sure that you understand what you're doing, before you attempt to do it

Also for future reference, before attempting channel open or closures always check the mempool, and pay high enough fee rates to avoid getting into these situations
